app.js

```diff
@@ -28,6 +33,20 @@ app.use((err, request, response, next) => {
     } next(err);
 });
 
+app.use((err, request, response, next) => {
+  if (err.code === "23503") {
```

Comment:
We can combine any psql errors into one error handler that checks the error codes